server/config/config.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Flow Playground
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+	"log"
+)
+
+// config holds all parsed environment variables
+var config struct {
+	envParsed  bool
+	platform   PlatformConfig
+	playground PlaygroundConfig
+	sentry     SentryConfig
+	database   DatabaseConfig
+}
+
+func GetPlatform() Platform {
+	if !config.envParsed {
+		parseConfig()
+	}
+	return config.platform.Type
+}
+
+func GetPlayground() PlaygroundConfig {
+	if !config.envParsed {
+		parseConfig()
+	}
+	return config.playground
+}
+
+func GetSentry() SentryConfig {
+	if !config.envParsed {
+		parseConfig()
+	}
+	return config.sentry
+}
+
+func GetDatabase() DatabaseConfig {
+	if !config.envParsed {
+		parseConfig()
+	}
+	return config.database
+}
```

Comment:
Are you opposed to using init() here since we only need one config struct ever? I like being able to use the envConfig interface for the sub-configs and calling the interface getConfig(). Otherwise each GetPlaygroundConfig, GetPlatformConfig, etc will have to do more work (right now they're a really nice one line that just takes in the environment variable prefix.